packages/components/src/public/SchemaQuery.test.ts

```diff
@@ -48,3 +48,38 @@ describe('resolveKeyFromJson', () => {
         );
     });
 });
+
+describe('parseSelectionKey', () => {
+    test('selectionKey with Schema, Query, and View', () => {
+        expect(SchemaQuery.parseSelectionKey('some-app-key|MySchema/MyQuery/MyView')).toEqual({
```

Comment:
this is likely for another issue / PR, but how does out parse selectionKey logic work for a queryName or viewName that has a | character in it?

Reply:
Definitely a question for a different PR. I would guess it doesn't handle it. This is why I tend to use stuff like `__snapshot__` as my separators.